styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -305,7 +314,9 @@ private void run() {
           throw CliExitException.of(ExitStatus.ApiError);
         }
       } else if (cause instanceof ClientErrorException) {
-        cliOutput.printError("Client error: " + cause.getMessage());
+        final Throwable rootCause = Throwables.getRootCause(cause.getCause());
```

Comment:
No need to cause.getCause() right?

Reply:
I meant `final Throwable rootCause = Throwables.getRootCause(cause.getCause());` -> `final Throwable rootCause = Throwables.getRootCause(cause);`

Reply:
I tried concatenating all the causes but that produced too much redundant and verbose messages for my taste, so I went straight for the important part, the root cause.